com.github.iwalton3.jellyfin-mpv-shim.json

```diff
@@ -0,0 +1,250 @@
+          ],
+          "config-opts": [
+            "--disable-static",
+            "--disable-bpf",
+            "--with-udevdir=/app/lib/udev"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://linuxtv.org/downloads/v4l-utils/v4l-utils-1.16.7.tar.bz2",
+              "sha256": "ee917a7e1af72c60c0532d9fdb9e48baf641d427aa7b009a9b2ca821f9e8e0d9"
+            }
+          ]
+        },
+        {
+          "name": "nv-codec-headers",
+          "cleanup": [
+            "*"
+          ],
+          "no-autogen": true,
+          "make-install-args": [
+            "PREFIX=/app"
+          ],
+          "sources": [
+            {
+              "type": "git",
+              "url": "https://git.videolan.org/git/ffmpeg/nv-codec-headers.git",
+              "commit": "cf8b0b2bb70b59068b06f1d5610627c8aa6d5652"
+            }
+          ]
+        },
+        {
+          "name": "ffmpeg",
+          "cleanup": [
+            "/include",
+            "/lib/pkgconfig",
+            "/share/ffmpeg/examples"
+          ],
+          "config-opts": [
+            "--enable-shared",
+            "--disable-static",
+            "--enable-gnutls",
+            "--disable-doc",
+            "--disable-programs",
+            "--disable-encoders",
+            "--disable-muxers",
+            "--enable-encoder=png",
+            "--enable-libv4l2",
+            "--enable-libdav1d"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://ffmpeg.org/releases/ffmpeg-4.2.2.tar.xz",
+              "sha256": "cb754255ab0ee2ea5f66f8850e1bd6ad5cac1cd855d0a2f4990fb8c668b0d29c"
+            }
+          ]
+        },
+        {
+          "name": "libass",
+          "cleanup": [
+            "/include",
+            "/lib/*.la",
+            "/lib/pkgconfig"
+          ],
+          "config-opts": [
+            "--disable-static"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/libass/libass/releases/download/0.14.0/libass-0.14.0.tar.xz",
+              "sha256": "881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2"
+            }
+          ]
+        },
+        {
+          "name": "uchardet",
+          "buildsystem": "cmake-ninja",
+          "config-opts": [
+            "-DCMAKE_BUILD_TYPE=Release",
+            "-DBUILD_STATIC=0"
+          ],
+          "cleanup": [
+            "/bin",
+            "/include",
+            "/lib/pkgconfig",
+            "/share/man"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://www.freedesktop.org/software/uchardet/releases/uchardet-0.0.6.tar.xz",
+              "sha256": "8351328cdfbcb2432e63938721dd781eb8c11ebc56e3a89d0f84576b96002c61"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "integration",

```

Comment:
Ideally all these files should be maintained in your repository and shipped as part of the tarball, although I'm not sure how feasible is it with pip.

Reply:
These files have been moved to the pip distribution.